docs/source/pages/DD2090.rst

```diff
@@ -46,6 +46,17 @@ But if you want to enable :ref:`Hardware message syncing` for all 4 cameras, you
     with dai.Device(config) as device:
         device.startPipeline(pipeline)
 
+Syncing with OAK-FFC-IR
+-----------------------
+
+If you have an `OAK-FFC-IR <https://shop.luxonis.com/collections/early-access/products/oak-ffc-ir>`__ compatible board (rev. R5M1E5 and up) and wish to use the IR module, you **must** first configure from which camera you want to have STROBE signal sent to the connector on the left (to which you connect the IR module). 
```

Comment:
Rephrase this; if user has FFC-4P R5 or higher, and would like to use OAK-FFC-IR.

> STROBE signal sent to the connector on the left (to which you connect the IR module).

`left` really depends on the orientation/perspective, I would rephrase this so it's not confused with LEFT camera.